config_samples/swarm/client.toml

```diff
@@ -0,0 +1,12 @@
+chain_id = "00000000-0000-0000-0000-000000000000"
+
+[account]
```

Comment:
I left `[account]`, because

```toml
[account]
id = "..." # account id

[authority]
account_id = "..." # authority's account id
```

Looks cumbersome, doesn't it?